### PR TITLE
fix: Larger radio clickbox

### PR DESCRIPTION
**Pull request by nicolls1**
In reference to thread in slack channel #frontend-dev

### CODE REVIEW

File: src/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -143,7 +146,11 @@ export const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
     };
 
     return (
-      <div className={clsx(classes.root, className)}>
+      <div
+        role="radio"
+        className={clsx(classes.root, className)}
+        onClick={() => inputRef?.current?.click()}
```
Comment (rob-snyder) [inline src/components/Radio/Radio.tsx:152]: Does this `onClick` sync to the onChange call in the `<input` tag through the `inputRef`?

**Review by rob-snyder — APPROVED** [review]
Looks good to me, though I'd defer to someone else's approval.

File: src/components/Radio/Radio.tsx (hunk 1 above)
Comment (nicolls1) [reply src/components/Radio/Radio.tsx:152]: Yup, it clicks the input element which triggers whatever callbacks the input has when clicked. So clicking this parent div is equivalent to clicking the input itself. 

Its common to have a setup like this for file inputs when you don't want to use the native file input display so you hide the file input and click it via ref when some other element is clicked.

File: src/components/Radio/Radio.tsx [hunk 2]
```diff
@@ -143,7 +146,11 @@ export const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
     };
 
     return (
-      <div className={clsx(classes.root, className)}>
+      <div
+        role="radio"
```
Comment (ynotdraw) [inline src/components/Radio/Radio.tsx:150]: I'm curious about having this here as well as on the `<input` on how a screen reader will behave? I can dig in a bit here later today

Comment (nicolls1) [reply src/components/Radio/Radio.tsx:150]: I added it because it was giving an error, but I'm not sure what best practice for this would be.

Comment (nicolls1) [reply src/components/Radio/Radio.tsx:150]: Digging in would be great!

**Pull request merged**